public/app/features/logs/components/LogRowMessage.tsx

```diff
@@ -72,6 +71,11 @@ const getStyles = (theme: GrafanaTheme2, showContextButton: boolean, isInDashboa
       visibility: hidden;
       width: ${showContextButton ? '80px' : '40px'};
     `,
+    logRowMenuCell: css`
+      position: absolute;
+      right: ${isInDashboard ? '40px' : `calc(75px + ${theme.spacing()} + ${showContextButton ? '80px' : '40px'})`};
```

Comment:
`75px` is coming from the `LogsNavigation` to the right of the table being 75px width. I agree it's not ideal, but it should work in most cases. Additionally I think this is always better that having the menu kinda "hidden" on long lines. 
I think this solves the issue for now, but we could dedicate some time bringing the context into a more stable shape in Q4.
WDYT?

Reply:
1 more question, is this space that is created by `theme.spacing()` wanted on purpose?
<img width="470" alt="image" src="https://user-images.githubusercontent.com/30407135/194067427-38801471-7610-4b85-b6b5-d7a81a851b74.png">

I am asking because without it, it ends at the same place as log line hover style. So I was just curious, if that is wanted.  
<img width="352" alt="image" src="https://user-images.githubusercontent.com/30407135/194067690-dee7a16b-62f5-47a3-b70a-6ed49e96f20d.png">